job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/BuildInDockerContext.groovy

```diff
@@ -1,6 +1,7 @@
 package javaposse.jobdsl.dsl.helpers.wrapper
 
 import javaposse.jobdsl.dsl.Context
+import javaposse.jobdsl.dsl.RequiresPlugin
 
 class BuildInDockerContext implements Context {
```

Comment:
The `@RequiresPlugin` annotation needs a `jobManagement` field to get access to Jenkins to determine the installed plugins. Extend `AbstractContext` to inherit the field.